src/components/Checkbox.tsx

```diff
@@ -18,9 +18,7 @@ export function CheckboxCell(props: CheckboxProps) {
   const note: NoteInfo = (cellProperties.row.original as any).__note__;
   /** state of cell value */
   const { contextValue, setContextValue } = useContext(CellContext);
-  const [checked, setChecked] = useState(
-    DataviewService.getDataviewAPI().value.isBoolean(contextValue.value)
-  );
+  const [checked, setChecked] = useState(contextValue.value);
```

Comment:
This change by itself could break the column type change option. Will be accepted and complemented with the actual parseLiteral of checkbox as boolean